tools/ci_connector_ops/ci_connector_ops/pipelines/bases.py

```diff
@@ -430,10 +430,16 @@ def build_include(self) -> List[str]:
         Returns:
             List[str]: List of directories or files to be mounted to the container to run a Java connector Gradle task.
         """
+        additional_inclusion = []
+        if self.context.connector.technical_name.endswith("-strict-encrypt"):
```

Comment:
Idea: it would be great if we had a more explicit mapping
```py
related_connectors = {
  "source-file-secure": ["source-file"],
  "source-postgres-strict-encrypt": ["source-postgres"]
  # ...
,}